src/Behat/MinkExtension/Context/MinkContext.php

```diff
@@ -425,8 +476,8 @@ public function printCurrentUrl()
 
     /**
      * Prints last response to console.
-     *
```

Comment:
Please don't remove these newlines separating the description from the step regex